### PR TITLE
[Behat][Admin][Shop] Change shop user password as administrator

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | fixes https://github.com/Sylius/Sylius/issues/6283
| License         | MIT

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
We should also remove unused tag (https://github.com/Sylius/Sylius/pull/5995) and unused methods in the listener.

File: features/user/managing_users/changing_shop_user_password.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    As an Administrator
+    I want to be able to change shop user password
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is a user "kibsoon@example.com" identified by "goodGuy"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Changing password of shop user
```
Comment (GSadee) [inline features/user/managing_users/changing_shop_user_password.feature]: Changing **a** password of **a** shop user?

File: features/user/managing_users/changing_shop_user_password.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        When I change user "kibsoon@example.com" password to "veryGoodGuy"
```
Comment (GSadee) [inline features/user/managing_users/changing_shop_user_password.feature]: When I change **a** user..?

**Comment by pjedrzejewski** [comment]
Thanks Mateusz!

**Pull request merged**